plugins/source/aws/policies_v1/views/metric_filters_with_active_subscriptions.sql

```diff
@@ -0,0 +1,16 @@
+-- describes metric alarms and pattern of alarm filter that has active subscriptions
+CREATE OR REPLACE VIEW view_aws_metric_filters_with_active_subscriptions AS
```

Comment:
This is never used?

Reply:
Oh. it was used in cloud watch checks.  it was used instad of `view_aws_log_metric_filter_and_alarm`. Which one do you think is more correct?

Reply:
@amanenk just so we know the reason why it's there. Not sure which is more correct, but we can keep it for now since it's already there.